development/extensions/modification_to_extension.rst

```diff
@@ -607,29 +600,29 @@ A complete file could look like this:
 Include extension's language files
 ==================================
 
-As the language files in your extension are not detected by the
+As the language files in your extension are not detected by
 ``$user->add_lang()`` any more, you need to use the ``$user->add_lang_ext()``
 method. This method takes two arguments, the first one is the fullname of the
 extension (including the vendor) and the second one is the file name or array of
-file names. so in order to load my newspage language file I now call:
+file names. So in order to load the Newspage language file we call:
 
 .. code-block:: php
 
      $user->add_lang_ext('nickvergessen/newspage', 'newspage');
 
-to load my language from
+to load the language file
 ``phpBB/ext/nickvergessen/newspage/language/en/newspage.php``
 
-File edits - Better don't edit anything, just use Events and Listeners
+File edits - Don't edit! Use Events and Listeners!
 ======================================================================
 
-As for the newspage Modification, the only thing that is now missing from
+As for the Newspage Modification, the only thing that is now missing from
 completion is the link in the header section, so you can start browsing the
-newspage.
+Newspage.
 
-In order to do this, I used to define the template variable in the
-``page_header()``-function of phpBB and then edit the ``overall_header.html``.
-But this is 3.1 so we don't like file edits anymore and added **events**
+In order to do this, the template variable used to be defined in the
+``page_header()`` function of phpBB along with an edit in the ``overall_header.html`` template.
+But this is not phpBB 3.0, so we don't like file edits anymore and now use **events**
```

Comment:
What about `... so we don't use file edits anymore and instead employ **events**`?